src-self-hosted/link.zig

```diff
@@ -88,17 +87,14 @@ pub fn writeFilePath(
 }
 
 pub fn openCFile(allocator: *Allocator, file: fs.File, options: Options) !File.C {
-    var self: File.C = .{
+    return File.C{
         .allocator = allocator,
         .file = file,
         .options = options,
-        .owns_file_handle = false,
```

Comment:
This is either incorrect or `openCFile` should be removed as available API

Reply:
Yeah, this shouldn't be public; it's only ever intended to be called (indirectly) from Module.init. The file should probably be closed and nulled after flushing.